nodes/analyzer/evaluate_image.py

```diff
@@ -133,11 +135,13 @@ def process(self):
         else: domV = self.domV
 
         # outputs
+        bw = [[]]
         red = [[]]
+        alpha = [[]]
         green = [[]]
         blue = [[]]
 
-        if outputs['R'].is_linked or outputs['G'].is_linked or outputs['B'].is_linked:
+        if outputs['R'].is_linked or outputs['G'].is_linked or outputs['B'].is_linked or outputs['A'].is_linked or outputs['BW'].is_linked:
```

Comment:
```python
if any(socket.is_linked for socket in self.outputs):
    ...
```

Reply:
That's magic :D 
Unfortunately I'll be on vacation for two weeks (Mexico with no PC). I will be able to handle that only at beginning of December. If you want to fix that in the meanwhile, fell free to modify it. Otherwise I'll take care of it when I'll be back.